golem-cli/src/clients/worker.rs

```diff
@@ -227,6 +243,59 @@ impl<C: golem_client::api::WorkerClient + Sync + Send> WorkerClient for WorkerCl
             .await?)
     }
 
+    async fn find_metadata(
+        &self,
+        template_id: RawTemplateId,
+        filter: Option<WorkerFilter>,
+        cursor: Option<u64>,
+        count: Option<u64>,
+        precise: Option<bool>,
+    ) -> Result<WorkersMetadataResponse, GolemError> {
+        info!(
+            "Getting workers metadata for template: {}, filter: {}",
+            template_id.0,
+            filter.is_some()
```

Comment:
Just curious, does filter not have a debug impl?

Reply:
yes, it have